Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -285,7 +243,7 @@ public final class RewardAddOnSelectionViewModel: RewardAddOnSelectionViewModelT
 
     let combinedPledgeTotal = Signal.combineLatest(
       additionalPledgeAmount,
-      allRewardsShippingTotal,
+      additionalPledgeAmount.mapConst(0),
```

Comment:
mapping this to a const of 0. explanation in [this comment](https://github.com/kickstarter/ios-oss/pull/2097/files#r1686789021)